k8s/tools/admin-node/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:18.04
```

Comment:
The `awscli` requires Python, so I was thinking that the following might be better?

```
FROM python:3
```
and then the install is a bit simpler as well (see comment below).